justfile

```
@@ -180,6 +180,10 @@ _api-install:
 @api-test docker_args="" tests="": _api-up
     docker-compose exec {{ docker_args }} web ./test/run_test.sh {{ tests }}
 
+# Run Django check command inside Docker
+@api-check docker_args="": _api-up
+    docker-compose exec {{ docker_args }} web python manage.py check
```

Could this be implemented in terms of the `dj` recipe?

```suggestion
@api-check docker_args="":
    just dj "{{ docker_args }}" check
```